Sources/NIO/Codec.swift

```diff
@@ -266,7 +273,8 @@ public class ByteToMessageHandler<Decoder: ByteToMessageDecoder> {
     internal private(set) var decoder: Decoder? // only `nil` if we're already decoding (ie. we're re-entered)
     private var state: State = .active
     private var removalState: RemovalState = .notBeingRemoved
-    private var buffer: B2MDBuffer = B2MDBuffer()
+    private var buffer: B2MDBuffer!
```

Comment:
Please add a comment that explains why this is an IUO.

Reply:
thanks, done